src/App.js

```diff
@@ -1,5 +1,17 @@
+import { useState } from 'react'
+import AddGamePage from './pages/AddGamePage'
+
 function App() {
-  return <div>Rob is der Geilste!</div>;
+  const [games, setGames] = useState([])
+  return (
+    <div>
+      <AddGamePage games={games} onSubmit={handleGamename} />
+    </div>
+  )
+
+  function handleGamename(name) {
```

Comment:
Maybe you could rename "name" to "newGame" ?

Reply:
Makes sense! Thank you. :)